dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/client/external/drc/DrcClient.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.laa.crime.dces.integration.client.external.drc;
```

Comment:
Do we need to package this deeply? Since not all of the new classes are external, this could cause confusion. It's just a client in the same way the maat one is, no?